src/current/v25.2/create-and-configure-changefeeds.md

```diff
@@ -53,46 +53,39 @@ For further detail on performance-related configuration, refer to the [Advanced
 - After you [restore from a full-cluster backup]({% link {{ page.version.version }}/restore.md %}#full-cluster), changefeed jobs will **not** resume on the new cluster. It is necessary to manually create the changefeeds following the full-cluster restore.
 - {% include {{ page.version.version }}/cdc/virtual-computed-column-cdc.md %}
 
-The following Enterprise and Core sections outline how to create and configure each type of changefeed:
+The following sections outline how to create and configure each type of changefeed:
```

Comment:
this is why i no longer ever say a number of anything, only ever "the following things ..." 🤣

Reply:
YES, ever since you mentioned this to me in another review, I'm trying to remove those where I notice. And "either"